pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -60,6 +61,9 @@ func RunClusterInfo(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command) error
 		return err
 	}
 	printService(out, "Kubernetes master", client.Host)
+	if len(client.AlternateHosts) > 0 {
```

Comment:
Since you added `Servers` to kubeconfig, don't forget to add it to `kubectl config view`